content/docs/rfcs/14/README.md

```diff
@@ -38,6 +38,10 @@ The `timestamp` field MAY be filled out to signify the time at which the message
 This field holds the Unix epoch time in nanoseconds. 
 Omitting it means the timestamp is unspecified.
 
+The `ephemeral` field MAY be filled out to signify the transient nature of the message.
```

Comment:
```suggestion
The `ephemeral` field MAY be set to `true` to signify the transient nature of the message.
```

Reply:
Addressed in [eeed3e2](https://github.com/vacp2p/rfc/pull/532/commits/eeed3e28fef460452ea7a4c58084d6ad7cf3f2e7)